### PR TITLE
Build debian packages for jsonc and memcached extensions

### PR DESCRIPTION
**Pull request by tmatsuo**
* Build memcached and jsonc extensions
* Added scripts for enabling/disabling extensions

supeceding #176

### CODE REVIEW

**Comment by tmatsuo** [comment]
@dazuma I wanted to add one more commit to #176 I prefer opening a new PR. I'd appreciate it if you could review this instead.

File: deb-package-builder/build.sh [hunk 1]
```diff
@@ -65,11 +68,39 @@ for FULL_VERSION in $(echo ${PHP_VERSIONS} | tr "," "\n"); do
...
+    envsubst '${SHORT_VERSION}' < debian/gcp-php.install.in > \
+             debian/gcp-php-${PHP_VERSION}.install
+    rm debian/gcp-php.install.in
+    envsubst '${SHORT_VERSION}' < debian/php-enmod.in > \
+             debian/php${SHORT_VERSION}-enmod
+    rm debian/php-enmod.in
+    chmod 755 debian/php${SHORT_VERSION}-enmod
+    envsubst '${SHORT_VERSION}' < debian/php-dismod.in > \
+             debian/php${SHORT_VERSION}-dismod
+    rm debian/php-dismod.in
```
Comment (dazuma) [inline deb-package-builder/build.sh]: Maybe just `rm debian/*.in debian/*/*.in` at the very end.

File: deb-package-builder/extensions/functions.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# common functions for building extensions
+
+E_PARAM_ERR=250
+
+download_from_pecl()
+{
+    # Download the source code, rename, extract it for debian package
+    if [ -z "$1" ]; then
+        return $E_PARAM_ERR
```
Comment (dazuma) [inline deb-package-builder/extensions/functions.sh]: Maybe `exit $E_PARAM_ERR` after printing an error message, if you want to catch this precondition? As it is, you're not using the return value.

File: deb-package-builder/build.sh (hunk 1 above)
Comment (tmatsuo) [inline deb-package-builder/build.sh]: Done

File: deb-package-builder/extensions/functions.sh (hunk 2 above)
Comment (tmatsuo) [inline deb-package-builder/extensions/functions.sh]: Done

**Comment by tmatsuo** [comment]
Thanks @dazuma PTAL!

**Pull request merged**